apps/api/src/server/app/middleware/error-handler.js

```diff
@@ -1,4 +1,5 @@
 import { validationResult } from 'express-validator';
+import pino from 'pino';
```

Comment:
do we not want to use the logger that you have defined in `apps/api/src/server/app/lib/logger.js`?